code-challenges.js

```diff
@@ -39,13 +72,33 @@ const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+//create a function named combine that takes two parameters x and y
+// const combine = (x,y) => {
+//   //concat will add 'y' the end of 'x'
+//   //.length will return the length of the combined arrays
+//   return x.concat(y).length
+  
+// }
+
+// console.log(combine(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns)) ---> 9
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Pseudo code:
 
 const currentCohort = "Bravo 2023"
 // Expected output: "3202 ovarB"
 
+//create a function that takes a string as a parameter - words
+// const backwards = (words) => { 
+//   // change <words> to an array with each character as an index
+//   //reverse the array
+//   //change the array back to a string
+//   return words.split('').reverse().join('')
```

Comment:
Nicely done! I like the pseudocode in between, if that's your style, go for it!👏🏽